manager/integration/tests/test_basic.py

```diff
@@ -85,7 +85,7 @@
 from common import create_host_disk, get_update_disks, update_node_disks
 from common import wait_for_disk_status, wait_for_rebuild_start, wait_for_rebuild_complete # NOQA
 from common import RETRY_BACKUP_COUNTS
-
+from common import LONGHORN_NAMESPACE
```

Comment:
Suggest keeping the blank line after common definitions.

Reply:
Updated. Thank you.